src/generic/FreeAssociativeAlgebraGroebner.jl

```diff
@@ -627,11 +644,41 @@ respect to this incomplete Groebner basis is `0`, it will also be `0` with respe
 
 If `remove_redundancies` is set to true, some redundant obstructions will be removed during the computation, which might save
 time, however in practice it seems to inflate the running time regularly.
+
+It can accept the keyword argument 'gb' as a precomputed partial Groebner basis.
```

Comment:
```suggestion
One can provide the keyword argument `gb` with a precomputed partial Groebner basis. 
It is assumed (but not checked) that all entries of `gb` are indeed contained in the ideal generated by `g`.
```
fix some formatting and add implicit assumption